### ISSUE TITLE
Does this library work with 64bit Android App? 

### ISSUE THREAD

**Issue by sleski** [silvaren/easyrs#15]
Because of render scrips files https://github.com/silvaren/easyrs/tree/master/easyrs/src/main/rs), does this lib work with Android 64bit App(arm64-v8a)?

**Comment by lenant** [comment]
Did not work for me, could not release the app in the store.

Bundletool was giving me warning:
WARNING: App Bundle contains 32-bit RenderScript bitcode file (.bc) which disables 64-bit support in Android. 64-bit native libraries won't be included in generated APKs.

**Comment by thanksmister** [comment]
Yeah, I have the same issue, I cannot create an APK with this library for 64 bit compatibility.  I don't think the androidx version of RenderScript plays nicely.   It seems to be related to the version of RenderScript (https://stackoverflow.com/questions/56356532/renderscript-and-playstore-64-bit-requirement/56477402).  Internally this library uses the support version of RenderScript android.support.v8.renderscript.   So your project would have to be reverted from androidX to use android.support.v8.renderscript instead or this library has to be updated to use androidx and also update the renderscriptTargetApi to at least 19, currently it is 16.
